02.calendar/cal.rb

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env ruby
+
+require 'optparse'
+require 'date'
+
+opt = OptionParser.new
+opt.on('-m')
+opt.on('-y')
+
+opt.parse!(ARGV)
+month = ARGV[0] || Date.today.mon
+year = ARGV[1] || Date.today.year
+
+header = "#{month}月"+" "+year.to_s
+print header.center(20) + "\n"
+
+day_of_week = %w(日 月 火 水 木 金 土)
+  print " "+day_of_week.join(" ") + "\n"
```

Comment:
```suggestion
print " "+day_of_week.join(" ") + "\n"
```

[nits] 不要なインデントを削除しましょう